management/clipper_manager.py

```diff
@@ -312,25 +312,19 @@ def get_app_info(self, name):
         Returns
         -------
         dict or None
-            Returns a dict with the application info if found. If the application
-            is not registered, None is returned.
+            Returns a dictionary with the specified application's info. This
+            will contain the attribute name-value pairs that were provided to
+            `register_application`. If no application with name `name` is
+            registered with Clipper, None is returned.
```

Comment:
Great, thank you for the catch; I was manually adding the name back in within the endpoint handling function without checking if redis actually returned a valid app object.